src/Microsoft/Silk.NET.Direct2D/Structs/D2D1CommandSink1VtblExtensions.gen.cs

```diff
@@ -138,18 +138,6 @@ public static unsafe int SetTextRenderingParams(this ComPtr<ID2D1CommandSink1> t
         return ret;
     }
 
-    /// <summary>To be documented.</summary>
-    public static int SetTextRenderingParams(this ComPtr<ID2D1CommandSink1> thisVtbl, ref IDWriteRenderingParams textRenderingParams)
```

Comment:
This is fine as this is new code but worth looking at. Same comment here about the circular dependency in IDWrite*

Reply:
this has been fixed by moving DWrite into D2D